pyproject.toml

```diff
@@ -17,6 +17,11 @@ classifiers = [
     "Operating System :: Microsoft :: Windows",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Do we really need to list all Python versions here? It feels to me like busy work to keep this updated for all our pyproject.toml (there are 3 other such files just in this repo)

Reply:
Hmm yes I am not sure either. These are the tags which will be shown at `Classifiers` on PyPI. We could also leave this out since the minimal version is fixed as well.